### PR TITLE
Add Kevin Gimbel

### PR DESCRIPTION
**Pull request by KevinGimbel**
**Add your name to verify:** Kevin Gimbel

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -8283,6 +8283,25 @@ module.exports = [
       'PHP',
       'SQL',
     ],
+  },{
+    name: 'Kevin Gimbel',
+    description: 'DevOps Engineer who loves Rust + web dev',
+    url: 'https://kevingimbel.de/uses',
+    twitter: '',
```
Comment (gatarelib) [inline src/data.js:8290]: "twitter" is not allowed to be empty

**Review by gatarelib — DISMISSED** [review]

File: src/data.js (hunk 1 above)
Comment (KevinGimbel) [reply src/data.js:8290]: Oh, so I can't join if I don't have a twitter account anymore?

Comment (BlakeCampbells) [reply src/data.js:8290]: Sorry for the delay, `twitter: '',` is invalid, but if you remove the line it is valid.

Comment (KevinGimbel) [reply src/data.js]: I see, I removed it thanks! :)

**Pull request merged**